Fix typo error in debian patch file:
0006-Fix-reachable-assert-in-SSLv2-servers.patch
--------------------------------------------------------------------

diff --git a/ssl/s2_srvr.c b/ssl/s2_srvr.c
index e297835..f350fc6 100644
--- a/ssl/s2_srvr.c
+++ b/ssl/s2_srvr.c
@@ -518,7 +518,7 @@ static int get_client_master_key(SSL *s)
 	/* If a bad decrypt, continue with protocol but with a
 	 * random master secret (Bleichenbacher attack) */
         if ((i < 0) || ((!is_export && i != EVP_CIPHER_key_length(c))
-                    || (is_export && i != ek))) {
+                    || (is_export && i != ek)))
 		{
 		ERR_clear_error();
 		if (is_export)
